src/ui/layout/qgslayoutitemwidgetbase.ui

```diff
@@ -728,27 +717,52 @@
         </item>
        </layout>
       </item>
+      <item row="0" column="0">
+       <widget class="QLabel" name="labelBlendMode">
+        <property name="text">
+         <string>Blending mode</string>
+        </property>
+       </widget>
+      </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelTransparency">
         <property name="text">
          <string>Opacity</string>
         </property>
        </widget>
       </item>
-      <item row="0" column="1">
-       <layout class="QHBoxLayout" name="horizontalLayout_3">
+      <item row="1" column="1">
+       <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
-         <widget class="QgsBlendModeComboBox" name="mBlendModeCombo"/>
+         <widget class="QgsOpacityWidget" name="mOpacityWidget" native="true">
+          <property name="focusPolicy">
+           <enum>Qt::StrongFocus</enum>
+          </property>
+         </widget>
         </item>
         <item>
-         <widget class="QgsPropertyOverrideButton" name="mBlendModeDDBtn">
+         <widget class="QgsPropertyOverrideButton" name="mOpacityDDBtn">
           <property name="text">
            <string>…</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
+      <item row="4" column="0">
+       <widget class="QLabel" name="label_5">
+        <property name="text">
+         <string>GeoPDF group</string>
```

Comment:
@nyalldawson , could we try to avoid being GeoPDF specific for this label (even though ATM it's every only used for GeoPDF)?

Reply:
@nirvn I've just removed the generic code now -- when I tried to make this generic it just ended up messy, as the requirements for eg SVG groups are very different to GeoPDF groups

Reply:
@nyalldawson , K, then GeoPDF-specific label there makes more sense.